packages/video_player/video_player/example/lib/main.dart

```diff
@@ -4,10 +4,6 @@
 
 // ignore_for_file: public_member_api_docs
 
-/// An example of using the plugin, controlling lifecycle and playback of the
```

Comment:
Should this comment be copied to be after the `imports`?

Reply:
This comment should definitely not be deleted.

Reply:
It was a mistake and fixed now.